recipes/krb5/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd src
+
+autoreconf -i
```

Comment:
@pelson I am using the GitHub tarball instead of the source URL in your PR. Is that OK?